src/pages/gateway/batching.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Batching with API Mesh
+description: Learn how API Mesh uses batching. 
+kewords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Batching with API Mesh for Adobe Developer App Builder
+
+Batching allows you to combine a group of requests into a single request, turning multiple queries into a single one. Compared to sending multiple queries simultaneously, batched requests result in better response times. They also avoid issues with rate-limiting.
```

Comment:
@revanth0212 : Should we also mention N+1 query problem like we have it in the graphql mesh documentation?